evdns.c

```diff
@@ -982,44 +982,48 @@ reply_run_callback(struct event_callback *d, void *user_pointer)
 {
 	struct evdns_request *handle =
 	    EVUTIL_UPCAST(d, struct evdns_request, deferred);
+	(void)user_pointer;
```

Comment:
Why not simply remove the argument?

Reply:
@azat do you mean changing `reply_run_callback`'s signature to `(struct event_callback *d, void *)`? I looked for similar examples in the existing code and found some in function declarations but not definitions, which used the `(void)var` pattern. e.g. `evsig_add()` in signal.c...

Reply:
Yes

Reply:
Oh, I see, it is required by the deferred callbacks, just ignore this comment.